docs/guide/teal-debugger.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: splash
+---
+
+# About TEAL Debugger
```

Comment:
```suggestion
# Debugging TEAL
```

Reply:
We use the top header (#) for the title. So there should be only one such header.

Reply:
Ok, thanks for info